src/main/java/ch/njol/skript/bukkitutil/PaperEntityUtils.java

```diff
@@ -114,8 +114,8 @@ public static void lookAt(LookAnchor entityAnchor, Object target, @Nullable Floa
 				Player player = (Player) entity;
 				if (target instanceof Vector) {
 					Vector vector = (Vector) target;
-					player.lookAt(vector.getX(), vector.getY(), vector.getZ(), LookAnchor.EYES);
-					player.lookAt(vector.getX(), vector.getY(), vector.getZ(), LookAnchor.FEET);
+					player.lookAt(player.getLocation().add(vector), LookAnchor.EYES);
+					player.lookAt(player.getLocation().add(vector), LookAnchor.FEET);
```

Comment:
Is this actually a correct change?
The docs for the parameter specify:
`The vector/location/entity to make the livingentity or player look at`
implying the vector is the position of the target from origin, whereas here you're adding that to the player's own position, implying it's relative to the player. 

If this method should be taking in a relative vector that needs to be updated in the docs for it, whereas if it should be taking in the absolute vector from origin then you should correct the places that are putting in a relative vector instead.

Reply:
Personally i think it makes more sense to have the vector be relative to the entity than to the origin but i am interested in what other think. I have noticed a few people (including myself) assume that looking at a vector means copying the pitch and yaw of the vector.

Reply:
Given the examples all indicate the intended result is for relative vectors (and that's how stuff like `push` works), I think this change could be considered a bug fix. It would be nice to clarify the docs a but more, though.